### PR TITLE
Sportsball

### PR DESCRIPTION
**Pull request by balderdash**

### CODE REVIEW

**Comment by Hamms** [comment]
What is `a.png` for?

File: apps/src/bounce/skins.js [hunk 1]
```diff
@@ -54,6 +72,31 @@ var CONFIGS = {
...
+      'basketball',
+      'football',
+      'hockey',
+      'soccer'
+    ],
+    paddles: [
+      'hand_1',
+      'hand_2',
+      'hockey',
+      'soccer'
```
Comment (Hamms) [inline apps/src/bounce/skins.js]: how difficult would it be to name these something like "foot" and "hockey_stick", for consistency?

File: apps/src/bounce/bounce.js [hunk 2]
```diff
@@ -1204,6 +1205,9 @@ Bounce.setBackgroundImage = function (backgroundUrl) {
 };
 
 Bounce.loadTiles = function (tiles, goalTiles) {
+  if (!skin.drawTiles) {
```
Comment (Hamms) [inline apps/src/bounce/bounce.js]: in what situation will `skin.drawTiles` be false?

**Comment by Hamms** [comment]
can the `apps/static/skins/basketball` directory be removed now?

**Comment by balderdash** [comment]
`a.png` was a test file, removed.

`basketball` and `sports` are still easier to handle as separate skins with their own directories, since the former includes the NBA team backgrounds, and the latter has the hockey/football/soccer assets.

 I spent a couple hours on Friday trying to get bounce tests running, but kept running into issues. I'll try a bit longer, but I'm probably going to merge this without tests if I can't get anything useful by the time staging unlocks. It's going to be used for an event on Friday.

File: apps/src/bounce/bounce.js (hunk 2 above)
Comment (balderdash) [reply apps/src/bounce/bounce.js]: Football, basketball, and soccer just have a simple background image, and don't need the red wall or separate goal tile images.

File: apps/src/bounce/skins.js (hunk 1 above)
Comment (balderdash) [reply apps/src/bounce/skins.js]: Not difficult, done.

**Pull request merged**